src/Documentation/sidebar.json

```diff
@@ -132,6 +132,17 @@
       {
         "label": "Anonymized Usage Analytics",
         "slug": "analytics"
+      },
+      {
+        "label": "How To",
+        "slug": "howto",
+        "source": "howto/index.md",
+        "children": [
```

Comment:
Do we plan to have more how to guides soon? This is probably a good idea but just checking whether there is an immediate need for a 3rd level for this one doc.

Reply:
> Do we plan to have more how to guides soon?

Sure. I am planning to add some more. You can also add some if you wish.

> This is probably a good idea but just checking whether there is an immediate need for a 3rd level for this one doc.

I think that the "How To" section is going to become a long one (having lots of items), and eventually it might need to be split from the User Guide as a top-level section on its own. For the time being it is Ok nested under the User Guide.

Reply:
Sounds good but I think it's best to create the 3rd level when more guides are added. What do you think @shcheklein?

Reply:
yes, I think we'll have to introduce a how to section soon. There are a few tickets that start with "How to ..." already - probably good candidates for that. May be we can move things like "Running on Windows" into this section>

Reply:
OK resolving this then. Thanks

Reply:
@jorgeorpinel this is the discussion around How to that happened a while ago ^^